doc/setup/install.rst

```diff
@@ -120,8 +127,35 @@ commands as root:
     # echo "deb http://nightly.odoo.com/9.0/nightly/deb/ ./" >> /etc/apt/sources.list
     # apt-get update && apt-get install odoo
 
-This will automatically install all dependencies, install Odoo itself as a
-daemon and automatically start it.
+You can then use the usual ``apt-get upgrade`` command to keep your installation up-to-date.
+
+**Enterprise:**
```

Comment:
we can also add (somewhere) in this page a link to "From Community to Enterprise" page